app/helpers/application_helper.rb

```diff
@@ -70,6 +70,9 @@ def locale_direction
   end
 
   def favicon_path
+    site_upload = SiteUpload.find_by(var: 'favicon')
```

Comment:
This could trigger spec error due to new update, If you could check the specs as well. Or else if you could write a case spec to validate what you are doing here.